### ISSUE TITLE
Tag Utterances with type information in Riff Server

### ISSUE THREAD

**Issue by jaedoucette** [rifflearning/zenhub-platform-project#217]
## Describe the ideal solution or feature request

Riff now has 3 utterance-based metrics (influence, affirmation, and interruption). 

Riff would like to relate utterance-based behavior to higher-level social attributes, like dominance, emotion, leadership, credibility, professionalism, confidence, and authenticity.

As a scientist, I would like to be able to extract a time-series of utterances from Riff-server, and train a machine learning model to predict higher-level social attributes from utterances. 

As a scientist, I would also like to be able to extract a time-series of utterances from Riff-server, and train a machine learning model to predict the tags on utterances from other utterance data, so that we can avoid the kind of arbitrary numbers we currently use.

## How does this tie into our current product?

Currently, all tags on utterance data are computed on the client's side, in the dashboard.

If we add an array of tags to the utterances in Riff-server, then we could avoid re-computing the tags every time a user views their dashboard, _and_ we could more easily perform machine learning tasks without duplicating code.

## Who asked for this?

John, for R&D purposes.

**Comment by brecriffs** [comment]
This is very intriguing!

**Comment by brecriffs** [comment]
Seems somewhat straightforward to do. Maybe could be done at the end of a meeting, since the utterance tags will never change? Maybe just have an array of tags attached to each utterance?

**Comment by jaedoucette** [comment]
@brecriffs Yeah, I kinda think so too. I think the main pieces of complexity will be in adjusting the feathers API.
